src/endpoints/account-memberships.js

```diff
@@ -0,0 +1,32 @@
+import CRUDExtend from '../extends/crud'
+
+class AccountMembershipsEndpoint extends CRUDExtend {
```

Comment:
If Create, Get and Delete are all available methods you don't need to extend from CRUDExtend. You can just define a simple class that doesn't extend anything. Take a look at `SettingsEndpoint`. By extending CRUDExtend class has all 4 CRUD methods available by default.